content/en/account_management/rbac/data_access.md

```diff
@@ -0,0 +1,183 @@
+* **Name dataset:** Restricted RUM data
+* **Select data to be included in this Dataset:**
+    * Telemetry type: RUM
+        * Filters: `@application.id:<rum-app-id>`
+* **Grant access:**
+    * Teams or roles of users who can see this RUM data      
+
+This configuration example would protect the RUM data from this application, and keep other data from this application available to existing users in your organization.
+
+#### Protecting all data from a service
+
+If you are instead looking to protect data from a specific service, you can build your Data Access Control configuration around the `service:` tag.
+
+For example, if you have a service `NewService` that is instrumented with Real User Monitoring (RUM) and capturing sensitive inputs from users, consider creating a Restricted Dataset only for that application:
+
+* **Name Dataset:** Restricted NewService data
+* **Select data to be included in this Dataset:**
+    * Telemetry type: RUM
+        * Filters: `@service:NewService`
+    * Telemetry type: Metrics
+        * Filters: `@service:NewService`
+    * Telemetry type: APM
+        * Filters: `@service:NewService`
+    * Telemetry type: Logs
+        * Filters: `@service:NewService`
+* **Grant access:**
+    * Team who owns the service
+
+This configuration example protects all supported data from `NewService`.
+
+### Teams and roles
+
+Data Access Control supports granting access to users through Datadog roles or teams. When granting access, consider your existing access control configuration and access strategy. If you are pursuing a service-based approach and are already [customizing the Service Catalog][9], take advantage of the service ownership model by using Teams as part of your Data Access Control configuration.
+
+**Note:** Teams used for Data Access Control must be configured such that adding or removing users can only be done by team members or an administrator, not `Anyone in the organization`.
+
+## Access enforcement
+
+Users in a Datadog organization with Data Access Control enabled can only see query results for data to which they have access, such as in a Dashboard, in an Explorer, or through the API. A restricted dataset removes access from the data bound in the restricted dataset from users who are not permitted across all Datadog experiences and entry points.
+
+### Data explorers
+
+When exploring Datadog with restrictions enabled, users without permissions are still able to cycle through the list of asset names (for example: applications or metrics), but they do not see query results, top tags, or facet details that have been restricted by restricted datasets. When querying a metric, for example, that contains data which is restricted from the user, a blank graph returns. It appears to the user as if their query does not match any data.
```

Comment:
```suggestion
When exploring Datadog with restrictions enabled, users without permissions can still browse the list of asset names (applications or metrics), but they cannot see query results, top tags, or facet details restricted by datasets. For instance, querying a metric with restricted data returns a blank graph, making it appear as if the query does not match any data.
```

Trying to remove the redundancy here with the word restricted in the same sentence :)